verification/curator-service/api/src/util/validate-env.ts

```diff
@@ -39,5 +40,9 @@ export default function validateEnv(): Readonly<{
             desc: 'URL to redirect to after the oauth consent screen',
             devDefault: 'http://localhost:3002/',
         }),
+        STATIC_DIR: str({
+            desc: 'directory to serve static files from',
+            default: '',
```

Comment:
Is this necessary?

Reply:
thanks that's actually a devDefault that I wanted here, in dev we use the reloadable react app and do not serve static files.